sstables/types.hh

```diff
@@ -593,7 +604,8 @@ struct scylla_metadata {
             disk_tagged_union_member<scylla_metadata_type, scylla_metadata_type::SSTableOrigin, sstable_origin>,
             disk_tagged_union_member<scylla_metadata_type, scylla_metadata_type::ScyllaBuildId, scylla_build_id>,
             disk_tagged_union_member<scylla_metadata_type, scylla_metadata_type::ScyllaVersion, scylla_version>,
-            disk_tagged_union_member<scylla_metadata_type, scylla_metadata_type::ExtTimestampStats, ext_timestamp_stats>
+            disk_tagged_union_member<scylla_metadata_type, scylla_metadata_type::ExtTimestampStats, ext_timestamp_stats>,
+            disk_tagged_union_member<scylla_metadata_type, scylla_metadata_type::SSTableGeneration, sstable_generation>
```

Comment:
I think this is confusing name, the sstable already has a generation, why is there a second one in the metadata?
I think sstable identifier would be a more appropriate name, that is what we are using this UUID for.

Reply:
makes sense, will change, thanks.